src/containers/Accounts/AMM/AMMAccounts/index.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useParams, useRouteMatch } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { AMMAccountHeader } from 'containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader'
+import { AccountTransactionTable } from 'containers/Accounts/AccountTransactionTable/index'
+import NoMatch from 'containers/NoMatch'
+import 'containers/Accounts/styles.scss'
+import { analytics, ANALYTIC_TYPES, formatAsset } from 'containers/shared/utils'
+import { Tabs } from 'containers/shared/components/Tabs'
+import { getAccountTransactions, getAMMInfo } from 'rippled/lib/rippled'
+import formatBalance from 'rippled/lib/txSummary/formatAmount'
+import SocketContext from '../../../shared/SocketContext'
+import { ERROR_MESSAGES } from '../../Errors'
+
+export interface AmmDataType {
+  balance: { currency: string; amount: number; issuer: string }
+  balance2: { currency: string; amount: number; issuer: string }
+  lpBalance: number
+  ammId: string
+  tradingFee: number
+  accountId: string
+  language: string
+}
+
+const getErrorMessage = (error: string) =>
+  ERROR_MESSAGES[error] || ERROR_MESSAGES.default
+
+export const AMMAccounts = (props: any) => {
+  const { id: accountId, tab = 'transactions' } = useParams<{
+    id: string
+    tab: string
+  }>()
+  const { path = '/' } = useRouteMatch()
+  const { t } = useTranslation()
+  const mainPath = `${path.split('/:')[0]}/${accountId}`
+  const rippledSocket = useContext(SocketContext)
+  const [data, setData] = useState<AmmDataType>()
+  const [error, setError] = useState<any>()
+  const { language } = props
+
+  useEffect(() => {
+    let asset1: { currency: string; issuer?: string }
+    let asset2: { currency: string; issuer?: string }
+    let ammData: any
+
+    /*
+    Get the first account transaction which in this case should be AMMInstanceCreate. From this we get
```

Comment:
```suggestion
    Get the first account transaction which in this case should be AMMCreate. From this we get
```
Transaction name has been renamed to `AMMCreate`